src/components/ToDoForm.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function ToDoForm({
+  textInput,
+  onTextChange,
+  onAddButtonClick,
+}) {
+  return (
+    <form>
+      <input type="text" placeholder="할 일을 입력해 주세요" onChange={onTextChange} value={textInput} />
```

Comment:
```suggestion
      <input 
        type="text" 
        placeholder="할 일을 입력해 주세요" 
        onChange={onTextChange} 
        value={textInput} 
      />
```
속성이 많을경우, 가독성을 고려해서 줄바꿈을 해주는것도 좋습니다 :)

Reply:
수정하였습니다!